corehq/apps/linked_domain/templates/linked_domain/get_started.html

```diff
@@ -0,0 +1,55 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<div id="ko-getting-started" data-bind="with: $root.gettingStartedViewModel">
+  <h3>{% trans "Linked Projects" %}</h3>
+  <p>
+    {% blocktrans %}
+      Get started by creating a link between two projects, allowing you to push and pull content.<br/>
+      If you want to pull content from another project into this project, make it a downstream project.<br/>
+      <a href="https://confluence.dimagi.com/display/commcarepublic/Linked+Project+Spaces" target="_blank">Learn more</a> about linked projects.
+    {% endblocktrans %}
+  </p>
+  <div class="spacer"></div>
+  <div class="row">
+    <!-- ko if: upstreamDomains().length > 0 -->
+      <div class="col-sm-6">
+        <h4>Add as Downstream Project</h4>
+        <p>
+          {% blocktrans %}
+            If you want to pull content from an upstream project, link this project to an upstream project.<br/>
```

Comment:
what's the `<br/>` here for? two things..usually try to avoid extra HTML in `{% blocktrans %}` unless absolutely needed. Secondly, could this be accomplished with a `<div class="spacer"></div>` after the `<p/>`s?

Reply:
Not sure why I included this.